src/components/TextField/SbTextField.vue

```diff
@@ -97,6 +97,7 @@
       <SbIcon
         v-if="showClearIcon"
         v-tooltip="{ label: 'Clear' }"
+        class="sb-textfield__icon--clearable"
```

Comment:
Nice work, Emanuel! Super quick ⚡ 
Any reason to use the class here instead of adding it to the `computedIconClasses`?

Reply:
Nice suggestion